attributes/default.rb

```diff
@@ -3,7 +3,7 @@
 default['snipeit']['doc_root'] = node['snipeit']['path'] + '/public'
 default['snipeit']['locale'] = 'en'
 default['snipeit']['timezone'] = 'US/Pacific'
-default['snipeit']['url'] = 'http://localhost'
+default['snipeit']['url'] = node['fqdn'] ? 'http://' + node['fqdn'] : 'http://' + node['hostname']
```

Comment:
Wonder if Ruby has something that adds this prefix. If not, might be useful to add a method to "webify" a string.